src/TicketService.java

```diff
@@ -16,7 +16,31 @@ public static void main(String[] args) {
                     false, 'A', 2.1));
         }
 
-        System.out.println(getTicketById("1111"));
+//        System.out.println(getTicketById("1111"));
```

Comment:
Always try to remove all unnecessary comments before push your changes to github